### DISCUSSION TITLE
Use Env variable for the verification redirect URL?

### DISCUSSION THREAD

**Discussion by brentrobbins** [nhost/nhost#99] [Q&A] [answered]
Can the verification redirect URL use an environment variable or can it only be set to that one value (for all environments)? Also can that only be changed in the nhost control panel or is it included in the config.yaml (client_url)?

**Comment by elitan** [comment, marked as answer]
For production you can set the redirect URL under Users -> Login settings -> Client URL.

For local development, you can update `auth_client_url`.

We'll also add support to add multiple allowed redirect URLs that can be used with the SDK, like:

```js
nhost.auth.signUp({email, password, options: { redirectTo: 'https://some-custom-domain.com' }});
```

But for now, you can only have a single "client URL" which will be the redirect URL being used.

I recommend that you use your production URL in the Nhost dashboard (e.g. `https://app.my-app.io`) and `http://localhost:3000` locally.

**Reply by brentrobbins** [reply]
Great! That works and glad to hear about the options parameters coming soon. Thank you @elitan.
